packages/note/src/react/index.js

```diff
@@ -0,0 +1,214 @@
+      glamor.css(css['.psds-note__contents']),
+      glamor.css(css[`.psds-note__contents.psds-theme--${themeName}`])
+    ),
+  footer: _ => glamor.css(css['.psds-note__footer']),
+  header: _ => glamor.css(css['.psds-note__header']),
+  heading: _ => glamor.css(css['.psds-note__heading']),
+  noteList: _ => glamor.css(css['.psds-note__list']),
+  metadata: themeName =>
+    glamor.compose(
+      glamor.css(css['.psds-note__metadata']),
+      glamor.css(css[`.psds-note__metadata.psds-theme--${themeName}`])
+    ),
+  metadataDatum: _ => glamor.css(css['.psds-note__metadata-datum']),
+  metadataDot: _ => glamor.css(css['.psds-note__metadata-dot'])
+}
+
+export default function Note(props) {
+  const themeName = useTheme()
+
+  const hasActions = !!props.actionBar && props.actionBar.length > 0
+  const hasAside = !!props.avatar
+  const hasHeading = !!props.heading
+  const hasMeta = !!props.metadata && props.metadata.length > 0
+
+  const actionBar = hasActions ? (
+    <ActionBar>
+      {props.actionBar.map((action, key) => {
+        return React.cloneElement(action, { key })
+      })}
+    </ActionBar>
+  ) : null
+
+  return (
+    <div {...styles.note(themeName, props)} {...filterReactProps(props)}>
+      {hasAside && (
+        <Aside>
+          {React.cloneElement(props.avatar, { size: Avatar.sizes.xSmall })}
+        </Aside>
+      )}
+
+      <Contents>
+        {hasHeading && (
+          <Header>
+            {props.heading && <Heading>{props.heading}</Heading>}
+            {actionBar}
+          </Header>
+        )}
+
+        {props.message}
+
+        <Footer>
+          {hasMeta && (
+            <Metadata>
+              {props.metadata.map((datum, i) => {
+                const hasNext = i < props.metadata.length - 1
+
+                return (
+                  <React.Fragment key={`datum-${i}`}>
+                    <MetadataDatum>{datum}</MetadataDatum>
+                    {hasNext && <MetadataDot />}
+                  </React.Fragment>
+                )
+              })}
+            </Metadata>
+          )}
+          {!hasHeading && actionBar}
+        </Footer>
+      </Contents>
+    </div>
+  )
+}
+
+function NoteList({ children, ...props }) {
+  return (
+    <ol {...styles.noteList()} {...filterReactProps(props)}>
+      {React.Children.map(children, child => (
+        <li>{child}</li>
+      ))}
+    </ol>
+  )
+}
+
+NoteList.displayName = 'Note.List'
+NoteList.propTypes = {
+  children: PropTypes.node
+}
+
+function ActionBar(props) {
+  const themeName = useTheme()
+  return <div {...styles.actionBar(themeName)} {...props} />
+}
+
+const Action = React.forwardRef((props, ref) => (
+  <button
+    ref={ref}
+    {...styles.action()}
+    {...filterReactProps(props, { tagName: 'button' })}
+  >
+    {props.icon}
+  </button>
+))
+
+Action.displayName = 'Note.Action'
+Action.propTypes = {
+  icon: elementOfType(Icon).isRequired,
+  title: PropTypes.string.isRequired
+}
+
+function AvatarLink(props) {
```

Comment:
What was the design thought around making a specialized `Note.AvatarLink` but not an equivalent `Note.TextLink` for the header or meta data (a la Card/Row), instead relying on them using the `<Link/>` package tag?  

Or did I misread what the design even is? :)

Reply:
`AvatarLink` was necessary for setting the avatar size...where a `TextLink` component wouldn't do anything differently than just the normal `Link`